mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/ui/HomeActivity.java

```diff
@@ -79,4 +82,15 @@ public void showClientDetails(Client client) {
 //                .buildRound(client.getName().substring(0, 1), R.color.colorPrimary);
 //        ivUserImage.setImageDrawable(drawable);
     }
+
+    @Override
+    public void onBackPressed() {
+        for (Fragment fragment : mHomeFragment.getChildFragmentManager().getFragments()) {
```

Comment:
Checking every fragment doesn't seems a great idea (performance wise). Could you think of some optimized solution.